Client.Core/Flux/Domain/FluxRecord.cs

```diff
@@ -26,7 +26,7 @@ public class FluxRecord
         /// <summary>
         /// The record's rows.
         /// </summary>
-        public List<KeyValuePair<string, object>> Rows { get; } = new List<KeyValuePair<string, object>>();
+        public List<KeyValuePair<string, object>> Row { get; } = new List<KeyValuePair<string, object>>();
```

Comment:
Please use `List<object>` without column names. The row should contains pure response from InfluxDB without additional data.

```suggestion
        public List<object> Row { get; } = new List<object>();
```